corehq/apps/app_manager/templatetags/xforms_extras.py

```diff
@@ -16,98 +17,136 @@ def translate(t, lang, langs=None):
             return t[lang]
 
 
+def tag_with_brackets(lang):
+    return ' [%s] ' % lang
+
+
+def tag_with_markup(lang):
+    style = 'btn btn-xs btn-info btn-langcode-preprocessed'
+    return format_html(' <span class="{}">{}</span> ', style, lang)
+
+
+def empty_tag(lang):
+    return ''
+
+
 @register.filter
-def trans(name, langs=None, include_lang=True, use_delim=True, prefix=False, escape=False):
+def trans(name, langs=None, include_lang=True, use_delim=True, prefix=False, strip_tags=False):
     langs = langs or ["default"]
     if include_lang:
         if use_delim:
-            tag = lambda lang: ' [%s] ' % lang
+            tag = tag_with_brackets
         else:
-            tag = lambda lang: '''
-                <span class="btn btn-xs btn-info btn-langcode-preprocessed">%(lang)s</span>
-            ''' % {"lang": html.escape(lang)}
+            tag = tag_with_markup
     else:
-        tag = lambda lang: ""
+        tag = empty_tag
```

Comment:
I know you didn't create the original name `tag` but for methods I feel using a clear action/verb improves clarity and this code has always been a little difficult to follow. Since `tag` can be a noun or a verb I feel like `format_tag` or `tag_translation` as a name would be better so it's clear that `tag` is a verb. Not blocking since this has always been the case.

Reply:
I think I got hung up on this as well. I'll try to put a better name in there